packages/block-library/src/page-list/index.php

```diff
@@ -88,28 +127,40 @@ function block_core_page_list_build_css_font_sizes( $context ) {
 /**
  * Outputs Page list markup from an array of pages with nested children.
  *
- * @param array $nested_pages The array of nested pages.
+ * @param array   $nested_pages The array of nested pages.
+ * @param array   $colors Color information for overlay styles.
+ * @param integer $depth  The nesting depth.
  *
  * @return string List markup.
  */
-function render_nested_page_list( $nested_pages ) {
+function render_nested_page_list( $nested_pages, $colors, $depth = 0 ) {
 	if ( empty( $nested_pages ) ) {
 		return;
 	}
 	$markup = '';
 	foreach ( (array) $nested_pages as $page ) {
-		$css_class = 'wp-block-pages-list__item';
+		$css_class       = 'wp-block-pages-list__item';
+		$style_attribute = '';
 		if ( isset( $page['children'] ) ) {
 			$css_class .= ' has-child';
 		}
-		$markup .= '<li class="' . $css_class . '">';
+
+		// If this is the first level of submenus, include the overlay colors.
+		if ( 1 === $depth ) {
+			$css_class .= ' ' . trim( implode( ' ', $colors['overlay_css_classes'] ) );
+			if ( '' !== $colors['overlay_inline_styles'] ) {
+				$style_attribute = sprintf( ' style="%s"', esc_attr( $colors['overlay_inline_styles'] ) );
+			}
+		}
+
+		$markup .= '<li class="' . $css_class . '"' . $style_attribute . '>';
```

Comment:
Is this working for folks? I'm not seeing the style attribute being added, so the submenus are inheriting from the navigation block.

<img width="1552" alt="Screen Shot 2021-05-18 at 1 59 03 PM" src="https://user-images.githubusercontent.com/1270189/118723299-1489d000-b7e2-11eb-8929-9562c1b49847.png">